### ISSUE TITLE
[total-functions/no-partial-division] false negatives

### ISSUE THREAD

**Issue by JustFly1984** [danielnixon/eslint-plugin-total-functions#726]
I have a code:

```
const pageSize = 50

const result = Math.ceil(
  someIntVariable / pageSize;
  // ^ here is false negative warning. pageSize is hardcoded, and can't be zero.
)

```

**Comment by danielnixon** [comment]
😉 https://github.com/danielnixon/eslint-plugin-total-functions/blob/master/src/rules/no-partial-division.ts#L43-L45

**Comment by danielnixon** [comment]
Published a new version with the fix. Let me know if there are still issues @JustFly1984 🙏

**Comment by JustFly1984** [comment]
Thank you, false errors are gone!
